### PR TITLE
Refactor QT UI variable names to be more consistent, Fix Difficulty default

### PR DESCRIPTION
**Pull request by barton2526**
Type of widget should always be in variable name

Always attempt to put widget type at end of name

Enforce consistent camelCase

Fix meta variable names (labelLabel)

Many UI variables (payTo, port, lang) were named too generically to be changed by me.  Someone with more expertise in the C++ side of the hooks will need to review which can be safely changed without breaking functionality.

----

Fix Difficulty defaulting to 1

Set to "N/A" like all other default labels

### CODE REVIEW

**Review by cyrossignol — APPROVED** [review]

**Review by jamescowens — APPROVED** [review]

File: src/qt/coincontroldialog.cpp [hunk 1]
```diff
@@ -78,18 +78,18 @@ CoinControlDialog::CoinControlDialog(QWidget *parent) :
...
+    ui->coinControlAfterFeeLabel->addAction(clipboardAfterFeeAction);
+    ui->coinControlBytesLabel->addAction(clipboardBytesAction);
+    ui->coinControlPriorityLabel->addAction(clipboardPriorityAction);
+    ui->coinControlLowOutputLabel->addAction(clipboardLowOutputAction);
+    ui->coinControlChangeLabel->addAction(clipboardChangeAction);
 
     // toggle tree/list mode
-    connect(ui->radioTreeMode, SIGNAL(toggled(bool)), this, SLOT(radioTreeMode(bool)));
-    connect(ui->radioListMode, SIGNAL(toggled(bool)), this, SLOT(radioListMode(bool)));
+    connect(ui->treeModeRadioButton, SIGNAL(toggled(bool)), this, SLOT(treeModeRadioButton(bool)));
```
Comment (barton2526) [inline src/qt/coincontroldialog.cpp:91]: there is a critical error in this PR here.  I believe the SLOT should not have been changed

**Comment by barton2526** [comment]
This PR currently results in 2 regressions on my local build:
1) As pointed out by @jamescowens the status label is permanently bolded for some reason (I cannot figure this out)
2) Clicking "inputs" in coin control crashes the wallet (I believe I see what is causing this).

Please back this PR out if possible and I will work to fix it.

**Comment by barton2526** [comment]
This change: https://github.com/gridcoin-community/Gridcoin-Research/commit/68361f2484d363f13d05425957aabfd11f351c5a fixes the bold status text.  It was a simple css change (thanks @cyrossignol !)

**Comment by barton2526** [comment]
See #1565 for fixes

**Pull request merged**